src/energyplus/ReverseTranslator/ReverseTranslateWindowPropertyFrameAndDivider.cpp

```diff
@@ -104,14 +104,14 @@ namespace energyplus {
       result.setDividerWidth(*d);
     }
 
-    d = workspaceObject.getDouble(WindowProperty_FrameAndDividerFields::NumberofHorizontalDividers);
-    if (d) {
-      result.setNumberOfHorizontalDividers(*d);
+    boost::optional<int> i = workspaceObject.getInt(WindowProperty_FrameAndDividerFields::NumberofHorizontalDividers);
+    if (i) {
+      result.setNumberOfHorizontalDividers(*i);
     }
 
-    d = workspaceObject.getDouble(WindowProperty_FrameAndDividerFields::NumberofVerticalDividers);
-    if (d) {
-      result.setNumberOfVerticalDividers(*d);
+    i = workspaceObject.getDouble(WindowProperty_FrameAndDividerFields::NumberofVerticalDividers);
```

Comment:
```suggestion
    i = workspaceObject.getInt(WindowProperty_FrameAndDividerFields::NumberofVerticalDividers);
```

@tijcolem this is the mistake I made that Windows is throwing on (which is actually helpful for once)

Reply:
Resolved in 81308adf2483eb6877d00d9e3c471f4701184d18 (after rebasing on develop)